xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -634,10 +651,20 @@
         <Parameter Name="item" Type="System.Text.RegularExpressions.Capture" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
       </Parameters>
       <Docs>
-        <param name="item">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="item">The object to remove from the collection.</param>
+        <summary>Removes the first occurrence of a specific object from the collection.</summary>
+        <exception cref="T:System.NotSupportedException">The collection is read-only.</exception>
+        <returns>
+          <see langword="true" /> if <paramref name="item" /> was successfully removed from the collection; otherwise, <see langword="false" />. This method also returns <see langword="false" /> if <paramref name="item" /> is not found in the original collection.</returns>
```

Comment:
Can't return anything if it throws in all cases :). Change this to say that NotSupportedException is thrown.